src/ui/AddressBook.qml

```diff
@@ -0,0 +1,173 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Controls.Material 2.12
+import QtQuick.Layouts 1.12
+import AddrsBookManager 1.0
+
+// Resource imports
```

Comment:
Also add commented the lines in case of we should need load delegates from the `qrc`